_ml-commons-plugin/extensibility/bedrock-connector.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: default
+title: Bedrock connector
+has_children: false
+nav_order: 63
+parent: Connectors
+grand_parent: Connecting to remote models 
+---
+
+# Bedrock connector
+
+To connect to a model on [Amazon Bedrock](https://aws.amazon.com/bedrock/), you can create a Bedrock connector. The following tutorial illustrates a [multimodal semantic search]({{site.url}}{{site.baseurl}}/search-plugins/neural-multimodal-search/) by text and image using a Bedrock model.
+
+## Prerequisites
+
+Before you start, update cluster settings with the following example configuration:
+
+```json
+PUT /_cluster/settings
+{
+  "persistent": {
+    "plugins.ml_commons.native_memory_threshold": 95,
+    "plugins.ml_commons.only_run_on_ml_node": false,
+    "plugins.ml_commons.allow_registering_model_via_url": true,
+    "plugins.ml_commons.model_access_control_enabled": true,
+    "plugins.ml_commons.trusted_connector_endpoints_regex": [
```

Comment:
Only this setting is mandatory to use bedrock connector. Can we remove other settings to keep it simple?